lib/taskana-core-test/src/test/java/acceptance/task/requestreview/RequestReviewWithBeforeSpiAccTest.java

```diff
@@ -1,5 +1,6 @@
 package acceptance.task.requestreview;
 
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
```

Comment:
Why are changes on this class not done with the previous commit?

Reply:
Good catch. This is a mistake.